keps/sig-network/2595-expanded-dns-config/README.md

```diff
@@ -356,6 +359,10 @@ The DNS lookup time can be increased, but it will be negligible.
 
 No
 
+- **Can enabling / using this feature result in resource exhaustion of some node resources (PIDs, sockets, inodes, etc.)?**
+
+No, this cannot result in resource exhaustion of node resources.
```

Comment:
is it because all lookup will be inside the container?

Reply:
Yes, setting a long DNS search list in the container only affects the process running inside the container.
Updated the answer.